<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelConsultas">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CONSULTAS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bRecorridoFecha">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RECORRIDOS POR FECHA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bRecorridoConductor">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RECORRIDOS POR CONDUCTOR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RecorridoUnidad">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RECORRIDOS POR UNIDAD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bRecorridoHoras">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RECORRIDOS POR HORAS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bRegresar">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>390</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>REGRESAR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
